### ISSUE TITLE
Non-continuous FEFunctions for high order FESpaces

### ISSUE THREAD

**Issue by fverdugo** [gridap/Gridap.jl#24]
@santiagobadia I have realized that high order fe spaces are not working properly. E.g.:

```julia
model = CartesianDiscreteModel(domain=(0.0,1.0,0.0,1.0), partition=(2,2))
order = 2
tags = [1,2,3,4,6,5]
fespace = ConformingFESpace(Float64,model,order,tags)
fun(x) = x[1] + x[2]
uh = interpolate(fespace,fun)
writevtk(trian,"trian", nref=4, cellfields=["uh"=>uh])

```

leads to a non-continuous FEFunction (here the model has 2x2 cells but I am visualizing with a finer mesh)

![non-continuous](https://user-images.githubusercontent.com/48508177/58192083-4255e000-7cc0-11e9-8338-74f3a0d02fe0.png)

Everything OK for the same example with `order = 1`

![continuous](https://user-images.githubusercontent.com/48508177/58192241-91037a00-7cc0-11e9-8f2f-77ff8f051950.png)

perhaps the problem is in the generation of the cell_eqclass.

**Comment by fverdugo** [comment]
We can discuss this tomorrow (among other topics)

**Comment by fverdugo** [comment]
Finally fixed via PR #56 

Closing issue.
